integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/BridgeResource.java

```diff
@@ -5,9 +5,9 @@
 import com.redhat.service.smartevents.infra.api.APIConstants;
 import com.redhat.service.smartevents.integration.tests.common.BridgeUtils;
 import com.redhat.service.smartevents.integration.tests.common.Constants;
-import com.redhat.service.smartevents.manager.api.models.requests.BridgeRequest;
-import com.redhat.service.smartevents.manager.api.models.responses.BridgeListResponse;
-import com.redhat.service.smartevents.manager.api.models.responses.BridgeResponse;
+import com.redhat.service.smartevents.manager.api.v1.models.requests.BridgeRequest;
```

Comment:
you might have conflicts with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1255 cc @sthundat